_config.yml

```diff
@@ -23,6 +23,10 @@ twitter_username: kaiachain
 github_username: kaiachain
 facebook_username: kaiachain
 medium_username: kaiachain
+repository: kaiachain/KIPS
```

Comment:
`KIPS`? Can I ask what this link is?

Reply:
I followed https://github.com/ethereum/EIPs/blob/master/_config.yml , but yeah i dont see this term in the jekyll library. will remove this.

Reply:
Removed this